src/loops.rs

```diff
@@ -69,6 +72,37 @@ impl LintPass for LoopsPass {
                 }
             }
 
+            // if this for-loop is iterating over a two-sided range...
+            if let ExprRange(Some(ref start_expr), Some(ref stop_expr)) = arg.node {
```

Comment:
This will catch `(10..0).rev()`, too.

We should ensure that the parent (use the parent method in `utils`) isn't a methodcall to `.rev()`, and add tests to ensure that these don't get linted